### PR TITLE
[Improvement][Logger]Logger server integrate into worker server

### PR DESCRIPTION
**Pull request by aaronlinv**
## Purpose of the pull request
Because Logger Server is a separate process, it increases the cost of deployment and maintenance, so integrate Logger server into Worker server and update the corresponding deployment scripts, Docker, K8s, etc

## Brief change log

Logger server integrate into Worker server
Remove the Logger Server from the deployment script, docker,docker-swarm,K8s

## Verify this pull request

This pull request is already covered by existing tests

### CODE REVIEW

**Comment by zhongjiajie** [comment]
It seem that you totally remove logger server. why are you going to do that? Did you missing something unexpect in your env, or just think logger server is not unnecessary.  I'm not very familiar with this code. Do you have time to look at it? @CalvinKirs @lenboo @caishunfeng

**Comment by caishunfeng** [comment]
Hi @aaronlinv , please resolve the conflicts, and add the issue for this pr, thxs.

**Comment by aaronlinv** [comment]
@caishunfeng Okay, I'll submit it later

**Comment by zhongjiajie** [comment]
> @caishunfeng Okay, I'll submit it later

Looking forward to this PR. you coould ping ShunFeng to review after you fix the conflict, we would review ASAP.

**Comment by aaronlinv** [comment]
@caishunfeng @zhongjiajie Because of the huge changes in 2.0 architecture, so I reworked and submitted the code, the deployment and documentation part I will submit the relevant commit after the merging, I hope to review and merge as soon as possible, so as to continue to follow up the subsequent iterations

**Comment by zhongjiajie** [comment]
I just approve to run PR CI, PTAL after CI green @caishunfeng

**Comment by caishunfeng** [comment]
PTAL @lenboo

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/LoggerServiceImpl.java
Comment (lenboo) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/LoggerServiceImpl.java:221]: magic number can be replaced with constants

Comment (lenboo) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/LoggerServiceImpl.java]: Magic number can be replaced with constants.

**Comment by chengshiwen** [comment]
@aaronlinv For the task **Condition**, **SubProcess**, and **Dependent**, integrating the **logger** into the **master** also needs to be considered. At the same time, do not concat the magic number 1234 or 5678, you should directly call `processInstance.getHost()` or `taskInstance.getHost()`

**Comment by aaronlinv** [comment]
@chengshiwen @lenboo Logger Server has been integrated into the Master, so that other tasks can view logs normally, and the IP and port are read directly by `TaskInstance`.

**Comment by zhongjiajie** [comment]
Please remember change our dev doc after this PR merged.

**Review by zhongjiajie — APPROVED** [review]
LGTM, Please remember change our dev doc after this PR merged.

**Review by caishunfeng — APPROVED** [review]
LGTM

**Review by lenboo — APPROVED** [review]
+1

**Comment by zhongjiajie** [comment]
@aaronlinv  I add label `miss document` for this PR, please change doc in https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/installation/pseudo-cluster.html dev dir. our website repo is in https://github.com/apache/dolphinscheduler-website

**Comment by chengshiwen** [comment]
@aaronlinv +1

**Pull request merged**
